src/hooks/useAuth.tsx

```diff
@@ -0,0 +1,8 @@
+import { useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import { selectCurrentUser } from '../store/reducers/AuthSlice';
+
+export const useAuth = () => {
+  const user = useSelector(selectCurrentUser);
+  return useMemo(() => ({ user }), [user]);
```

Comment:
🧐
Блоки констант, условия, `return`-ы и обособленные куски логики лучше отбивать пустой строкой
```suggestion
  const user = useSelector(selectCurrentUser);
  
  return useMemo(() => ({ user }), [user]);
```

Reply:
Не понял вопрос!